app/controllers/api/v1/cookie_and_jwt_api/protocol_subscriptions_controller.rb

```diff
@@ -11,6 +11,20 @@ def show
           render json: @protocol_subscription, serializer: Api::ProtocolSubscriptionSerializer
         end
 
+        # Specify either a start_date or a start_time. A given start_date takes precedence.
+        # If a start_time is specified (and no start_date is specified),
+        # the start_date is chosen as the given start_time added to midnight the next day.
+        def create
+          result = SubscribeToProtocol.run!(
+            protocol_name: protocol_subscription_create_params[:protocol_name],
+            person: current_user,
+            start_date: start_date,
+            only_if_not_subscribed: true
+          )
+          SendInvitations.run
```

Comment:
is this necessary? Can't we just wait for the job to run? (this is fine too, but I like the idea that you don't have to do this manually)

Reply:
Fixed.